File: src/main/java/io/stouder/adonis/edge/editor/comments/EdgeCommenter.java
package io.stouder.adonis.edge.editor.comments;

import com.intellij.lang.Commenter;
import org.jetbrains.annotations.Nullable;

public class EdgeCommenter implements Commenter {
    @Override
    public @Nullable String getLineCommentPrefix() {
        return null;
    }

    @Override
    public @Nullable String getBlockCommentPrefix() {
        return "{{--";
    }

    @Override
    public @Nullable String getBlockCommentSuffix() {
        return "--}}";
    }

    @Override
    public @Nullable String getCommentedBlockCommentPrefix() {
        return null;
    }

    @Override
    public @Nullable String getCommentedBlockCommentSuffix() {
        return null;
    }
}

File: src/main/java/io/stouder/adonis/edge/editor/comments/EdgeCommenter.kt
package io.stouder.adonis.edge.editor.comments

import com.intellij.lang.Commenter

class EdgeCommenter : Commenter {
    override fun getLineCommentPrefix(): String? {
        return null
    }

    override fun getBlockCommentPrefix(): String? {
        return "{{--"
    }

    override fun getBlockCommentSuffix(): String? {
        return "--}}"
    }

    override fun getCommentedBlockCommentPrefix(): String? {
        return null
    }

    override fun getCommentedBlockCommentSuffix(): String? {
        return null
    }
}
